src/perfEvents_linux.cpp

```diff
@@ -494,7 +495,7 @@ long PerfEvents::_interval;
 Ring PerfEvents::_ring;
 CStack PerfEvents::_cstack;
 
-int PerfEvents::createForThread(int tid) {
+int PerfEvents::createForThread(pid_t tid) {
```

Comment:
It's `int` in the header file; I'd change type in either place for consistency.

Reply:
Nicer as `pid_t`, I suppose. So I'll make use of this option and change it everywhere :)